Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml

```diff
@@ -12,8 +12,7 @@ http://www.springframework.org/schema/aop     http://www.springframework.org/sch
         The beans available in this file must be identified in the "description" element for the Spring configuration application. Beans are listed between braces
```

Comment:
Updating a config file seems way out of scope of this issue.  Is this a separate fix?